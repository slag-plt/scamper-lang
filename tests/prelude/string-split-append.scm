(string-split "HelloWorld" "w")
(string-split "Hello woww World" "w")
(string-split "" "")
(string-split "00110011" "1")
(string-split "1234567" "5")
(string-append "HelloWorld" "Hello woww World" )
(string-append " " "00110011" "1234567" )
(string-append "1234567" "89101112" "13141516")