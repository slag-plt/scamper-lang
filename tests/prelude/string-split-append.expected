(list "HelloWorld")
(list "Hello " "o" "" " World")
null
(list "00" "" "00" "" "")
(list "1234" "67")
"HelloWorldHello woww World"
" 001100111234567"
"12345678910111213141516"
